src/plugins/cosmos/components/AssetClaimCard/AssetClaimCard.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Text as CText } from '@chakra-ui/react'
+import BigNumber from 'bignumber.js'
+import osmosis from 'assets/osmosis.svg'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card, CardProps } from 'components/Card/Card'
+
+type AssetClaimCardProps = {
+  assetSymbol: string
+  assetName: string
+  cryptoRewardsAmount: BigNumber
+  fiatRewardsAmount: BigNumber
+  renderButton?: () => JSX.Element
+}
+
+export const AssetClaimCard = ({
+  assetSymbol,
+  assetName,
+  cryptoRewardsAmount,
+  fiatRewardsAmount,
+  renderButton,
+  ...styleProps
+}: AssetClaimCardProps & CardProps) => (
```

Comment:
I know we have some inconsistencies across the app with this, some I even added myself when crunching the previous iteration of modals, but it would be great to always define the intersection of `OwnProps` & `InjectedProps` in the component type itself 
```suggestion
}: AssetClaimCardProps) => (
```